blog/good-compliance-programs-do-not-require-fancy-platforms.md

```diff
@@ -0,0 +1,235 @@
+There is no course that he knows of that teaches this.
+
+And yet, the more you learn, the more it becomes apparent, at least to me, that what makes a compliance audit easy, versus a slogging and demoralizing haze, is not whether you have fancy automation tools, a fancy web ui, or an all-in-one auditing platform, its whether your company has embraced the framework against which you wish to be compliant and understands how your operations fit into it.
+Thats it.
+You either know what you are doing, or you don't.
+And the unfortunate truth is that many of these platforms take advantage of those who don't know what they are doing and sell them a set of tools that are still only really effective if you know what you are doing.
+
+What these platforms could (and should) be doing is teaching.
+
+## Keys to a successful compliance program
+
+I have now been through two SOC2 audits with Temporal Technologies.
+Throughout each of them I acted as the compliance lead, a sort of project manager.
+My primary responsibilities were to organize materials, gather evidence for audits, schedule meetings, and generally create a structure to operate in.
+I was also the auditor's point of contact with our company, funneling and prioritizing communications, and translating requirements into action items for our team.
+And I made sure to keep other key players up-to-date, or to push on them when deadlines closed in.
+
+I have worked with both [Strike Graph](https://www.strikegraph.com/), one of the compliance platforms, and [Martin Cozzi](https://www.marana.io/), an independent personalized consultant.
+
+From these experiences, here are my 5 key things that shaped our successful compliance program.
+
+### 1. Retain the help of an expert to get ramped up
+
+Unfortunately there is not a lot of room for trial and error when it comes to compliance programs, and a lot of pitfalls to overcome through self study.
+So, I definitely recommend getting the help of someone with experience.
+
+For startups and other small companies that do not have in-house experience yet, platforms like Strike Graph (for SOC2 at least) are actually not a bad way to go.
+They can help you get up and going for a lot less money than a personalized consultant.
+Strike Graph for example maintains a team of customer success agents who can steer you in the right direction by providing a lot of context and boilerplate policies for your organization to utilize.
+
+But just remember they are not in the business of understanding your business, and therefore they most likely will not be able to help with precision custom Control Statements that scale for you.
+And they may not be able to help accurately translate the expectations of your auditing partner into action steps.
+
+Also, something that all compliance platforms suffer from is that using an additional third party system means that it is yet another external system to login to and learn how to use.
+Think about it.
```

Comment:
If you're writing an intro/educational article, stay away from things like "Think about it". It can come off as condescending.